pages/linux/systemd-analyze.md

```diff
@@ -9,3 +9,11 @@
 - Print a tree of the time critical chain of units:
 
 `systemd-analyze critical-chain`
+
+- Create an SVG file showing when each system service started, highlighting the time that they spent on initialization:
+
+`systemd-analyze plot > {{path/to/file.svg}}`
+
+- Plot a dependency graph and convert it to an SVG file:
+
+`systemd-analyze dot | dot -T{{svg}} > {{path/to/file.svg}}`
```

Comment:
I have no idea in what way the output of that command should be usable, since it's impossible to follow all the arrows, but it defenetly looks cool. 😄